frontend/catalyst/compiler.py

```diff
@@ -358,11 +358,13 @@ def get_default_flags(options):
         if platform.system() == "Linux":
             file_path_within_package = "../scipy.libs/"
             file_extension = ".so"
+            rpath_flag = "-Wl,--disable-new-dtags"
```

Comment:
Minor: we could add these to the system flags below instead of creating a new one. Let's also add a comment what this flag does (I've already forgotten 😆 ).

Reply:
Yes. It makes sense to move there. I have also added comments for that.